nominees/wildlife-watcher.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Wildlife Watcher",
+  "aliases": [
+    "Wētā Watcher"
+  ],
+  "description": "An open-source wildlife camera that records animals and uses AI to identify them.",
+  "website": "https://www.wildlife.ai/projects/weta-watcher/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/wildlifeai/wai_data_tools/blob/main/LICENSE.txt"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 15,
+      "evidenceURL": "https://www.wildlife.ai/projects/weta-watcher/"
```

Comment:
The `evidenceURL` is not instructive as to how this work relates to SDG 15. Could we ask them to share copy that connects Wildlife Watcher to some of the Goal Targets and Indicators?

Perhaps they could start with these ones:

![Screenshot from the UN SDGs website. Pictured are Targets 15.4 and 15.5, along with each of their indicators. https://sdgs.un.org/goals/goal15](https://user-images.githubusercontent.com/4721034/176590217-6ae66151-8a57-4919-8dbf-6e4f050b75d7.png "Screenshot from the UN SDGs website. Pictured are Targets 15.4 and 15.5, along with each of their indicators. https://sdgs.un.org/goals/goal15")

Reply:
Followed-up with Wildlife Watcher team on the above, will update here as soon as we hear from them. Thanks.